lib/bulbs-elements/util/get-analytics-manager.js

```diff
@@ -9,22 +9,16 @@ FIXME: videohub-player depends on there being an instance of our analytics manag
         3. Have all sites follow a convention for where AnalyticsManager
            lives.
 */
-/* global avclubAnalytics, onionan, clickholean, starwipe */
+/* global starwipe */
 
 export default function () {
-  let analyticsManager;
-  if (window.avclubAnalytics) {
-    analyticsManager = avclubAnalytics;
+  if (window.starwipe) {
```

Comment:
pr [here](https://github.com/theonion/starwipe/pull/291) to consolidate this further